### ISSUE TITLE
Redesign: Need design for field selects

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#8264]
Need a design to determine exactly what they should look like.

For example on macOS they look completely wrong:
![image](https://user-images.githubusercontent.com/1190097/51764074-65191480-2091-11e9-93bb-b77cb71231fb.png)

On windows they look like this:
![image](https://user-images.githubusercontent.com/1190097/51764734-22583c00-2093-11e9-872c-af26a6320788.png)

**Comment by jryans** [comment]
Isn't this a duplicate of #8249? (The proposed fix in https://github.com/matrix-org/matrix-react-sdk/pull/2502 hasn't merged.)

**Comment by turt2live** [comment]
Nope, https://github.com/matrix-org/matrix-react-sdk/pull/2502 doesn't address how the fields are outright wrongly represented on macOS (see image).

**Comment by jryans** [comment]
Ah okay, I wasn't sure how far we wanted to go in terms of controlling style across UAs. (As mentioned before, there's no design spec for `<select>` currently...)

**Comment by turt2live** [comment]
Yea, I'll adopt this issue as the canonical "figure out selects" issue

**Comment by ara4n** [comment]
@turt2live points out that implementing all the accessibility stuff with a non-native select is a pain in the ass, and it turns out that we *do* have enough CSS these days to override the look & feel of the native select control on webkit & firefox:

https://jsfiddle.net/pxjkqru9/

If someone (@jryans? @turt2live?) can apply it to the selects to make them non-fugly it'd be appreciated (or i may have time later today to do it, but would rather it wasn't blocked on me)

**Comment by jryans** [comment]
I'll take a look.

**Comment by jryans** [comment]
Fixed by https://github.com/matrix-org/matrix-react-sdk/pull/2540.
